docs/static/ls-to-cloud.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[connecting-to-cloud]]
+=== Send {ls} Output to Elastic Cloud
+
+Our hosted Elasticsearch Service is available on AWS, GCP, and Azure.
+{ess-trial}[You can try the Elasticsearch Service for free].
+
+Logstash comes with two settings that simplify sending output to
+https://cloud.elastic.co/[Elastic Cloud]: `cloud.id` and `cloud-auth`. 
+You can set these values in the `logstash.yml` file or from the command line.
+
+==== Cloud ID
```

Comment:
This looks like it's just for setting up modules with ESS, which feels like a lower priority given the current state of modules. Would it make sense to broaden this scope to sending data to ESS with cloud_id and cloud_auth in the ES output instead?

https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-cloud_id

Reply:
Info is already there.